README.md

```diff
@@ -23,6 +24,229 @@ Terra Toolkit is a utility module used to facilitate independent development of
+* **enabled** - Flat to disable selenium docker; useful for CI environments which can startup the docker instance outside of test runs. Defaults to true
+* **cleanup** - Destroy the docker container after the test run. Defaults to false.
+* **image** - The docker image to use for test runs. Defaults to `selenium/standalone-chrome` or `selenium/standalone-firefox` based on browser capabilities specified in config.
+
+#### Example
+```js
+// wdio.conf.js
+const wdioConf = require('terra-toolkit/wdio/conf');
+const localIP = require('ip');
+
+const staticServerPort = 4567;
+
+const config = {
+  baseUrl: `http://${localIP.address()}:${staticServerPort}`,
+  seleniumDocker: {
+    // Disable if running in Travis
+    enabled: !process.env.TRAVIS,
+  },
+  ...wdioConf.config,
+};
+
+exports.config = config;
+```
+
+
+### Accessibility Tests
+Terra toolkit automatically includes a wdio-axe-service which enhances an WebdriverIO instance with commands for accessibility testing using the [Axe](https://github.com/dequelabs/axe-core) utility.
+
+#### Options
+
+Under the key `axe` in your wdio.conf.js you can pass a configuration object with the following structure:
+
+* **inject** - True if the axe script should be injected by the test running. Disable if axe is already included in the test files which slightly speed up runs. Defaults to true.
+
+#### Example
+```js
+// wdio.conf.js
+const wdioConf = require('terra-toolkit/wdio/conf');
+const localIP = require('ip');
+
+const staticServerPort = 4567;
+
+const config = {
+  baseUrl: `http://${localIP.address()}:${staticServerPort}`,
+  axe: {
+    // Don't inject axe script, its included in test files
+    inject: false,
+  },
+  ...wdioConf.config,
+};
+
+exports.config = config;
+```
+
+
+#### Usage
+
+`browser.axe([{options}]);`
+
+The following options are available:
+
+* **viewports**:
+  An array of viewports `{ width, height }` to run the accessibility test in. If none provided uses the current viewport.
+* **rules**:
+  The axe rules configuration to test. See [axe documentation](https://www.axe-core.org/docs/)
+* **runOnly**:
+  The axe tags to filter the validations to run on the accessibility to test. See [axe documentation](https://www.axe-core.org/docs/)
+* **context**:
+  A css selector to scope the accessibility test to. See [axe documentation](https://www.axe-core.org/docs/)
+
+#### Examples
+```js
+it('ignores inaccessibility based on rules', () => {
+  browser.url('/inaccessible-contrast.html');
+  const rules = {
+    'color-contrast': { enabled: false },
+  };
+  expect(browser.axe({ viewports, rules })).to.be.accessible();
+});
+
+it('runs only specified tags', () => {
+  browser.url('/inaccessible-text.html');
+  const runOnly = {
+    type: 'tag',
+    values: ['color-contrast'],
+  };
+  expect(browser.axe({ viewports, runOnly })).to.be.accessible();
+});
+
+it('runs only specified context', () => {
+  browser.url('/inaccessible-contrast.html');
```

Comment:
Do you need to direct to a specific URL when providing this specific context? Also, this feels more like content than context.

Reply:
Context is following the `axe` api naming convention for what they describe as scoping tests to an element.